qa/L0_java_sequence_batcher/SequenceTest.java

```diff
@@ -0,0 +1,636 @@
+            "unable to serialize model status protobuf");
+
+        JsonParser parser = new JsonParser();
+        JsonObject model_metadata = null;
+        try {
+          model_metadata = parser.parse(buffer.limit(byte_size.get()).getString()).getAsJsonObject();
+        } catch (Exception e) {
+          FAIL("error: failed to parse model metadata from JSON: " + e);
+        }
+
+        FAIL_IF_ERR(
+            TRITONSERVER_MessageDelete(model_metadata_message),
+            "deleting status protobuf");
+
+        if (!model_metadata.get("name").getAsString().equals(model_name)) {
+          FAIL("unable to find metadata for model");
+        }
+
+        boolean found_version = false;
+        if (model_metadata.has("versions")) {
+          for (JsonElement version : model_metadata.get("versions").getAsJsonArray()) {
+            if (version.getAsString().equals("1")) {
+              found_version = true;
+              break;
+            }
+          }
+        }
+        if (!found_version) {
+          FAIL("unable to find version 1 status for model");
+        }
+
+        FAIL_IF_ERR(
+            ParseModelMetadata(model_metadata, is_torch_model),
+            "parsing model metadata");
+      }
+
+      // Create the allocator that will be used to allocate buffers for
+      // the result tensors.
+      TRITONSERVER_ResponseAllocator allocator = new TRITONSERVER_ResponseAllocator(null);
+      FAIL_IF_ERR(
+          TRITONSERVER_ResponseAllocatorNew(
+              allocator, responseAlloc, responseRelease, null /* start_fn */),
+          "creating response allocator");
+
+      // Inference
+      TRITONSERVER_InferenceRequest irequest = new TRITONSERVER_InferenceRequest(null);
+      FAIL_IF_ERR(
+          TRITONSERVER_InferenceRequestNew(
+              irequest, server, model_name, -1 /* model_version */),
+          "creating inference request");
+
+      FAIL_IF_ERR(
+          TRITONSERVER_InferenceRequestSetId(irequest, "my_request_id"),
+          "setting ID for the request");
+
+      FAIL_IF_ERR(
+          TRITONSERVER_InferenceRequestSetReleaseCallback(
+              irequest, inferRequestComplete, null /* request_release_userp */),
+          "setting request release callback");
+
+      // Inputs
+      String input0 = is_torch_model[0] ? "INPUT__0" : "INPUT";
+
+      long[] input0_shape = {1};
+
+      int datatype = TRITONSERVER_TYPE_INT32;
+
+      FAIL_IF_ERR(
+          TRITONSERVER_InferenceRequestAddInput(
+              irequest, input0, datatype, input0_shape, input0_shape.length),
+          "setting input 0 meta-data for the request");
+
+      String output0 = is_torch_model[0] ? "OUTPUT__0" : "OUTPUT";
+
+      FAIL_IF_ERR(
+          TRITONSERVER_InferenceRequestAddRequestedOutput(irequest, output0),
+          "requesting output 0 for the request");
+
+      long correlation_id = 5;
+      int iterations = 9;
+      int expected_result = 45;
```

Comment:
where do these values come from? Can they be input params?

Reply:
Correlation_id doesn't really matter, it just needs to be a non-zero number so that the sequence of requests has an ID tying them together.

As for iterations and expected result, I made them match sequence_batcher_test.py in terms of setting a value of 9 and 45. I can switch them to input parameters if that's better... it doesn't make a big difference, as these models essentially sum 1 to n. In this case, 1+2+...+8+9 = 45. 

I renamed these variables and added comments for clarity. Let me know if that works. If you still want me to switch any of these to input params, let me know. I don't think it'd affect what we're testing in the sequence batcher, but it's an easy change.